mmv1/products/securityposture/PostureDeployment.yaml

```
@@ -22,6 +22,7 @@ description: |
 references:
   guides:
     'Create and deploy a posture': 'https://cloud.google.com/security-command-center/docs/how-to-use-security-posture'
+  api: 'https://cloud.google.com/security-command-center/docs/reference/securityposture/rest/v1/PostureDeployment'
```

We generally link to the Overview page for the resource methods rather than the type page:

```suggestion
  api: 'https://cloud.google.com/security-command-center/docs/reference/securityposture/rest/v1/organizations.locations.postureDeployments'
```